language/English/strings.po

```diff
@@ -3321,7 +3321,143 @@ msgctxt "#802"
 msgid "%s of %s available"
 msgstr ""
 
-#empty strings from id 803 to 849
+#. Message in a dialog when a user wants to delete a timer that is part of a repeating schedule
+#: xbmc/pvr/windows/GUIWindowPVRTimers.cpp
+#: xbmc/pvr/windows/GUIWindowPVRBase.cpp
+#: xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp
+msgctxt "#803"
+msgid "Do you also want to delete all other timers from this repeating schedule?"
+msgstr ""
+
+#. Label of spinner control for choosing the timer type (once, weekdays, mondays,..)
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#804"
+msgid "Schedule type:"
+msgstr ""
+
+#. Type selection for an advanced (epg)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#805"
+msgid "Once"
+msgstr ""
+
+#. Type selection for an advanced (epg)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#810"
+msgid "Weekdays (Mon-Fri)"
+msgstr ""
+
+#. Type selection for an advanced (epg)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#811"
+msgid "Weekends (Sat-Sun)"
```

Comment:
I'm not sure we can solve this with pure labels in a sane way. Core would require new settings in our Language&Region section where one could alter his week layout. So for the time being I don't think we can do much. I'd only use "Weekdays" and "Weekends" though, so that we don't have to change the labels again onc we support this.
@Montellese do you think we could add something like a week layout to our language system? We might need two settings - "First day of week" (Monday or Sunday), "Weekend days" (select list with Mon-Sun). Eventually also a preferred "Short day format" (2-char, 3-char) as well as "Short month format" (2-char, 3-char) which could be advanced/expert level and ofc a utility class/functions to get accordingly formated values.